src/components/Books.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+
+function Books({ title, author }) {
+  return (
+    <div className="book-container">
+      <h2 className="book-title">{title}</h2>
+      <p className="book-author">{author}</p>
+      <button type="button" className="remove-btn">Remove</button>
+    </div>
+  );
```

Comment:
- [ ] This `<Books>` component ideally should render a list of books and is not expected to take the `title` and `author` props. The props would be needed in the singular `<Book>` component which should render the title and the author. Please make this modification to your component 👍 
- [ ] Please render your `<addBook>` component here so that the form to add a new book is displayed 🙏